Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10-alpine
+
+RUN pip3 install requests
+
+COPY . /opt/sapcli
+
+RUN cd /opt/sapcli &&\
+	python3 -m pip install -r requirements.txt &&\
+	chmod +x /opt/sapcli/sapcli &&\
+	echo '#!/bin/sh' > /usr/bin/sapcli &&\
+	echo 'python3  /opt/sapcli/sapcli "$@"' >> /usr/bin/sapcli &&\
+	chmod +x /usr/bin/sapcli
+
+WORKDIR /var/tmp
+
+ENTRYPOINT ["sapcli"]
```

Comment:
Or you can do this `ENTRYPOINT ["/usr/bin/python3", "/opt/sapcli/bin/sapcli"]`

Reply:
I like this one the most.

Reply:
Having the script in `/usr/bin/` allows to run scripts in the container where you can use `sapcli` as a global command, that was the main idea here.

What do you think about adding `/opt/sapcli/bin` to `PATH`? I guess that would achieve the same result without copying or symlinking files around.

Reply:
OK, add it to PATH.

PS: I plan to pack sapcli in Fedora/Red Hat rpm package. We could also introduce Alpine package.